runtime/vara/src/migrations.rs

```diff
@@ -22,11 +22,10 @@ use crate::*;
 pub type Migrations = (
     BagsListMigrate<Runtime>,
     pallet_gear_bank::migrations::MigrateToV1<Runtime>,
+    pallet_gear_builtin::migration::MigrateToV1<Runtime, TreasuryAccount>,
```

Comment:
We will top up Treasury, right?

Reply:
Treasury has enough funds on both networks to payout from 3 to 4 tokens